webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -16,8 +16,8 @@
                 <a href="{{ url(
                     'crashstats:topcrasher',
                     product=query['product'],
-                    versions=query['versions'],
-                    date_range_type='report',
+                    versions=';'.join(query['versions']),
```

Comment:
Yes, and it breaks on prod already: https://crash-stats.mozilla.com/topcrashers/?product=Firefox&version=%E2%81%BB (that's a ``NoReverseMatch`` error, because the unicode symbol is not supported in the old topcrasher app's URL). But that only happens if a user manually enters an incorrect character here, so I guess we can leave that as is. What do you recommend?